app/templates/courses/teacher-dashboard-nav.pug

```diff
@@ -33,4 +33,6 @@
           li
             a.track-click-event(href=`${ozBaseURL()}/teachers/professional-development`, data-event-action='Teacher Nav Click PD')
               small.label(data-i18n='teacher_dashboard.pd_short')
-
+          li 
```

Comment:
can you actually add it before 'View Outcomes Report' -> 'Use New Dashboard' in smaller font than outcome reports?
Is it possible we can create a link which takes the user to 'Try the new Teacher Dashboard' section on click by using HTML id? Only if it's small effort?